rgd/geodata/urls.py

```diff
@@ -16,17 +16,17 @@
     # Temporary redirect for home page
     path(r'', RedirectView.as_view(url='geodata/spatial_entries/', permanent=False), name='index'),
     path(
-        'geodata/rasters/<int:pk>/',
+        'geodata/raster/<int:pk>/',
         views.RasterEntryDetailView.as_view(),
         name='raster-entry-detail',
     ),
     path(
-        'geodata/fmv_entries/<int:pk>/',
+        'geodata/fmv/<int:pk>/',
         views.FMVEntryDetailView.as_view(),
         name='fmv-entry-detail',
     ),
     path(
-        'geodata/geometries/<int:pk>/',
+        'geodata/geometry/<int:pk>/',
```

Comment:
This had been bugging me for a while. It's a small change with little consequence so I included it here rather than making a separate PR. If there is debate on the URLs in general, we should still merge this and make a new issue/PR to clean up these URLs